### ISSUE TITLE
errors when grabbing this part in MAB.

### ISSUE THREAD

**Issue by BararQ** [KerboNerd/KNI-HP-CYREX#6]
GameData/KerboNerdLaboratories/Parts/Engines/scifiVortex/vpart/VRTX-3
I get these errors when grabbing this part in MAB.

Failed to find primary and secondary engine, check engine IDs and module definitions
[ModuleSrfFX]: No IThrustProvider module found at index 3!

**Comment by KerboNerd** [comment]
Yeah.. I'm still working on this engine. Since it uses non-stock plugins, it runs into errors very often.
